lib/Dialect/TTIR/IR/TTIROpsInterfaces.cpp

```diff
@@ -5,12 +5,44 @@
 #include "ttmlir/Dialect/TTIR/IR/TTIROpsInterfaces.h"
 #include "ttmlir/Dialect/TTIR/IR/TTIR.h"
 
+#include <cstdint>
+#include <llvm/ADT/SmallVector.h>
+#include <mlir/Dialect/Traits.h>
+#include <mlir/IR/BuiltinAttributes.h>
+#include <mlir/IR/Operation.h>
 #include <mlir/Support/LogicalResult.h>
 
 mlir::LogicalResult
 mlir::tt::ttir::detail::verifyElementwiseOp(mlir::Operation *op) {
```

Comment:
Does this mean that the shape will change within ttir dialect to reflect broadcasted shapes?

Reply:
No it won't change, here we just verify operands are of compatible shapes.